pkg/ipcache/ipcache.go

```diff
@@ -522,7 +522,7 @@ func (ipc *IPCache) UpsertMetadataBatch(updates ...MU) (revision uint64) {
 	ipc.metadata.Lock()
 	for _, upd := range updates {
 		if !upd.IsCIDR || ipc.metadata.prefixRefCounter.Add(upd.Prefix) {
-			prefixes = append(prefixes, ipc.metadata.upsertLocked(upd.Prefix, upd.Source, upd.Resource, upd.Metadata...)...)
+			prefixes = append(prefixes, ipc.metadata.upsertLocked(upd.Prefix, upd.Source, ipcacheTypes.ResourceID("consolidate-"+upd.Prefix.String()), upd.Metadata...)...)
```

Comment:
```suggestion
			resource := upd.Resource
			if upd.IsCIDR {
				resource = ipcacheTypes.ResourceID("consolidate-"+upd.Prefix.String())
			}
			prefixes = append(prefixes, ipc.metadata.upsertLocked(upd.Prefix, upd.Source, resource, upd.Metadata...)...)
```

Should this be something closer to ^ to properly handle the non-isCIDR cases like the name manager operations?

Reply:
Fixed and added a test case!